daemon/src/daemon.rs

```diff
@@ -115,14 +115,15 @@ impl DocumentActor {
         base_dir: PathBuf,
         init: bool,
         is_host: bool,
+        persist: bool,
     ) -> Self {
         // If there is a persisted version in base_dir/.ethersync/doc, load it.
         // TODO: Pull out ".ethersync" string into a constant.
         let persistence_file = base_dir.join(".ethersync/doc");
         let persistence_file_exists = sandbox::exists(&base_dir, &persistence_file)
             .expect("Could not check for the existence of the persistence file");
 
-        let crdt_doc = if persistence_file_exists && !init {
+        let crdt_doc = if persistence_file_exists && !init && persist {
```

Comment:
Tiny refactoring suggestion could be to pull this out to it's own variable. Just because it getting longer and it'd be more "speaking" that way without having to look into the implementation of the branch.
```suggestion
        let load_crdt_doc = persistence_file_exists && !init && persist;
        let crdt_doc = if load_crdt_doc {
```